ReactAndroid/src/main/java/com/facebook/react/devsupport/DevSupportManagerImpl.java

```diff
@@ -23,13 +23,16 @@
 import android.os.AsyncTask;
 import android.util.Pair;
 import android.widget.Toast;
+
 import com.facebook.common.logging.FLog;
 import com.facebook.debug.holder.PrinterHolder;
 import com.facebook.debug.tags.ReactDebugOverlayTags;
 import com.facebook.infer.annotation.Assertions;
 import com.facebook.react.R;
+import com.facebook.react.ReactInstanceManager;
```

Comment:
this creates a circular dependency 

`//java/com/facebook/catalyst/js/react-native-github/ReactAndroid/src/main/java/com/facebook/react:react -> //java/com/facebook/catalyst/js/react-native-github/ReactAndroid/src/main/java/com/facebook/react/devsupport:devsupport -> //java/com/facebook/catalyst/js/react-native-github/ReactAndroid/src/main/java/com/facebook/react:react`

maybe you should leave the interface in react/devsupport?